src/backends/github/implementation.js

```diff
@@ -73,13 +73,15 @@ export default class GitHub {
         sem.take(() => this.api.readFile(file.path, file.sha).then((data) => {
           resolve({ file, data });
           sem.leave();
-        }).catch((err) => {
+        }).catch((err = true) => {
           sem.leave();
-          reject(err);
+          console.error(`failed to load file from GitHub: ${file.path}`);
```

Comment:
@erquhart @Benaiah I added a console error for now, we can add a toast notification or other GUI action later.